app/src/main/java/com/github/ybecker/epforuml/QuestionDetailsActivity.kt

```diff
@@ -45,20 +48,27 @@ class QuestionDetailsActivity : AppCompatActivity() {
 
         // retrieve cache value
         cache = intent.getParcelableArrayListExtra("savedQuestions")!!
+        answersCache = intent.getParcelableArrayListExtra("savedAnswers")!!
+        comingFromFragment = intent.getStringExtra("comingFrom")!!
```

Comment:
Maybe you could declare constants for the string instead of using the same string at different places